prac_08/silver_service_taxi_test.py

```diff
@@ -1,8 +0,0 @@
-from prac_08.silver_service_taxi import SilverServiceTaxi
-
-taxi = SilverServiceTaxi("Silver Service Taxi", 100, 2)
-taxi.drive(18)
```

Comment:
I don't see the .drive method in the SilverServiceTaxi class so not sure what this line of code is doing but I haven't tried to run the code and haven't done the prac yet so ignore if just blind i.e. Prac07 had 'def drive(self, distance):' in the car.py file but I don;t see this in the SilverServicceTaxi code?